### ISSUE TITLE
Custom scope question not being triggered

### ISSUE THREAD

**Issue by njncalub** [jvanbruegge/commithelper#3]
Hi, sorry for filing another ticket. I'd be happy to send a PR if that's okay.

## Current Behavior

I've set `commithelper.allowCustomScopes` to `true`. However, this input-type question is not triggered when selecting `custom` as a scope:

```typescript
// prompt.ts
{
	type: 'input',
	name: 'scope',
	message: 'Enter a custom scope:',
	when: answers => answers.scope === 'custom',
},
```

```bash
$ pnpm commit
? Select the type of change that you're committing: feat:  A new feature
? What is the scope of this change (e.g. component or file name): custom
? Write a short, imperative tense description of the change (max 87 chars):
 (0)
```

## Expected Behavior

```bash
$ pnpm commit
? Select the type of change that you're committing: feat:  A new feature
? What is the scope of this change (e.g. component or file name): custom
? Enter a custom scope: my-custom-scope
? Write a short, imperative tense description of the change (max 78 chars):
 (0)
```

### package.json

```json
"commithelper": {
  // ...
  "allowCustomScopes": true,
  // ...
},
```

## Possible Solutions

There are 2 questions with the `name` of `scope` and it looks like it's clashing with each other.

```typescript
// prompt.ts
{
    type: 'list',
    name: 'scope',
    message: `What is the scope of this change (e.g. component or file name):`,
    choices: getChoices(config),
    when: answers => getChoices(config)(answers).length > 0,
},
{
    type: 'input',
    name: 'scope',
    message: 'Enter a custom scope:',
    when: answers => answers.scope === 'custom',
},
```

The second one can be changed to `customScope`. Afterwards, the rest of the code can be updated accordingly:

```typescript
// prompt.ts
{
    type: 'input',
    name: 'customScope',
    message: 'Enter a custom scope:',
    when: answers => answers.scope === 'custom',
},
```

```typescript
// message.ts
export interface Message {
    type: string;
    scope?: string;
    customScope?: string;
    subject: string;
    body?: string;
    breaking?: string;
    issuesClosed?: string;
}

// Update the rest of the code.
```

**Comment by jvanbruegge** [comment]
You are right, I've fixed this behavior. No need to change the message interface though

**Comment by jvanbruegge** [comment]
I forgot to set `askAnswered` as documented here: https://github.com/SBoudrias/Inquirer.js#questions

**Comment by jvanbruegge** [comment]
I released version 1.2.0 with both fixes

**Comment by njncalub** [comment]
That was amazingly fast. Thank you! :grinning:
